generators/entity/index.js

```diff
@@ -62,8 +62,15 @@ module.exports = yeoman.generators.Base.extend({
     // don't prompt if data are imported from a file
     if (this.entityConfig.useConfigurationFile == true &&  this.entityConfig.data && typeof this.entityConfig.data.enableEntityAudit !== 'undefined') {
       this.enableAudit = this.entityConfig.data.enableEntityAudit;
+
+      if (typeof this.config.get('auditFramework'); !== 'undefined') {
```

Comment:
Also we can check db type as well to set default if we cant find from yo rc file ?